### ISSUE TITLE
'The engine "node" is incompatible with this module' as of 2.4.0

### ISSUE THREAD

**Issue by misha** [Vuepic/vue3-date-time-picker#65]
**Describe the bug**

Was just updating to latest to grab a bug fix and came across the following error when installing via `yarn`:

```
$ yarn
yarn install v1.22.15
[1/4] Resolving packages...
[2/4] Fetching packages...
error vue3-date-time-picker@2.4.0: The engine "node" is incompatible with this module. Expected version ">=14". Got "12.22.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Previous versions do not appear to produce this error.

**To Reproduce**
1. Add `2.4.0` to `package.json`.
2. Install with `yarn`, although probably `npm` works too.

**Expected behavior**
Should install the latest version.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser: N/A
 - Version: N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**
`2.3.5` appears to install correctly. I have pinned my version to that dependency for now.

**Comment by Jasenkoo** [comment]
Sry, I changed in package.json node engine to 14 since some tests fail on v12. Will see and add it back, but keep in mind that other libraries are already on v14+. If I add it back, it will be until the end of the maintenance period (2022-04-30) and after that, it will be dropped.

**Comment by Jasenkoo** [comment]
Fix [v2.4.1](https://github.com/Vuepic/vue3-date-time-picker/releases/tag/v2.4.1) is now live
